lib/resources/r4/related_person.yaml

```diff
@@ -147,8 +147,9 @@ fields:
     }
   note: |
     <ul>
-      <li>The <code>relationship.coding</code> must contain exactly one proprietary system.</li>
-      <li>The <code>relationship.coding</code> cannot contain a standard system.</li>
+      <li> The <code>relationship.coding</code> must contain exactly one proprietary system. </li>
+      <li> The <code>relationship.coding</code> cannot contain a standard system. </li>
```

Comment:
nit: do we need space here? I don't see spaces at the beginning and end of <li> and </li> before the change

Reply:
seems like we do need space here. my bad